genesis_block.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
Do we want this file in this repo?

Reply:
it everyone has to have the genesis block, it is not easier to have the genesis in the repo?

Reply:
what do you thin @aesedepece